src/mlpack/core/util/program_doc.cpp

```diff
@@ -20,36 +21,70 @@ using namespace mlpack::util;
 using namespace std;
 
 /**
- * Construct a ProgramDoc object.  When constructed, it will register itself
+ * Construct a ProgramName object.  When constructed, it will register itself
  * with IO.  A fatal error will be thrown if more than one is constructed.
  *
  * @param defaultModule Name of the default module.
- * @param shortDocumentation A short two-sentence description of the program,
+ */
+ProgramName::ProgramName(const std::string& programName)
```

Comment:
Just realized the documentation isn't correct, `@param defaultModule Name of the default module.` do you mind to fix it here as well?

Reply:
Fixed in the last commit.